pkg/minikube/cni/cni.go

```diff
@@ -37,14 +37,19 @@ import (
 const (
 	// DefaultPodCIDR is the default CIDR to use in minikube CNI's.
 	DefaultPodCIDR = "10.244.0.0/16"
-)
 
-var (
+	// DefaultCNIConfDir is the default CNI Config Directory path
+	DefaultCNIConfDir = "/etc/cni/net.d"
```

Comment:
this doesn't seem to be used anywhere outside this package, could it be private ?

Reply:
good catch, thanks! i'll fix that

Reply:
actually, since i've used it to check if the current CNIConfDir is different than the DefaultCNIConfDir - in order to update the crio conf, it has to stay public